docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -9,286 +12,421 @@ Storm Reference - Filtering
+  use full ``YYYY/MM/DD hh:mm:ss.mmm`` syntax. Synapse allows you to use either lower resolution
   values (e.g., ``YYYY/MM/DD``) or wildcard values (e.g., ``YYYY/MM*``). In particular, wildcard syntax
   can be used to specify any values that match the wildcard expression. See the type-specific documentation
   for :ref:`type-time` types for a detailed discussion of these behaviors.
 
-Filters Using Standard Comparison Operators
--------------------------------------------
 
-Filter operations can be performed using any of the standard mathematical / logical comparison operators
-(comparators):
+.. _filter-prop-std-primary:
 
-- ``=``: equals (described above)
-- ``!=`` : not equals
-- ``<`` : less than
-- ``>`` : greater than
-- ``<=`` : less than or equal to
-- ``>=`` : greater than or equal to
+Filter by Primary Property Value
+++++++++++++++++++++++++++++++++
 
 **Syntax:**
 
-*<query>* **+** | **-** *<form>* | *<prop>* *<comparator>* *<valu>* | *<pval>*
+*<query>* **+** | **-** *<form>* *<operator>* *<valu>*
 
-**Examples:**
 
-*Filter by Not Equals:*
+*Filter the current working set to exclude the loopback IPv4 address (127.0.0.1):*
 
-- Filter results to exclude the domain ``google.com``:
+.. storm-pre:: [ inet:ipv4=127.0.0.1 inet:ipv4=8.8.8.8 ] -inet:ipv4 = 127.0.0.1
+::
+  
+  <query> -inet:ipv4 = 127.0.0.1
 
-.. storm-pre:: inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com +inet:fqdn != google.com
+.. storm-pre:: [ inet:ipv4=127.0.0.1 inet:ipv4=8.8.8.8 ] +inet:ipv4 != 127.0.0.1
 ::
-    
-    <query> +inet:fqdn != google.com
+  
+  <query> +inet:ipv4 != 127.0.0.1
+
+
+.. _filter-prop-std-secondary:
 
+Filter by Secondary Property Value
+++++++++++++++++++++++++++++++++++
 
-*Filter by Less Than:*
+**Syntax:**
+
+*<query>* **+** | **-** [ *<form>* ] **:** *<prop>* *<operator>* *<pval>*
 
-- Filter results to include only WHOIS records collected prior to January 1, 2017:
 
-.. storm-pre:: [inet:whois:rec=(wellsfargo.com, 2019/11/24 03:30:07.000) :created="1993/02/19 05:00:00.000"] +inet:whois:rec:asof < 2017/01/01 
+*Filter the current working set to include only those domains (inet:fqdn nodes) that are also logical zones:*
+
+.. storm-pre:: [ inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com ] +inet:fqdn:iszone = 1
 ::
-    
-    <query> +inet:whois:rec:asof < 2017/01/01
+  
+  <query> +inet:fqdn:iszone = 1
 
 
-.. storm-pre:: inet:whois:rec +:asof < 2017/01/01
+.. storm-pre:: [ inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com ] +:iszone = 1
 ::
-    
-    <query> +:asof < 2017/01/01
+  
+  <query> +:iszone  = 1
+
+*Filter the current working set to exclude any files (file:bytes nodes) with a PE compiled time of 1992-06-19 22:22:17:*
+
+.. storm-pre:: [ (file:bytes=sha256:2d168c4020ba0136cd8808934c29bf72cbd85db52f5686ccf84218505ba5552e :mime:pe:compiled="1992/06/19 22:22:17.000") ] -file:bytes:mime:pe:compiled = '1992/06/19 22:22:17'
```

Comment:
is this example supposed to be creating an additional file:bytes node that will pass the filter?